doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md

```diff
@@ -282,9 +282,10 @@ As seen above, the default configuration distributed with the image is tied to c
 * PostgreSQL:
     * It only works for building historical context data in PostgreSQL.
     * The endpoint for PostgreSQL is `iot-postgresql` but can be changed through the CYGNUS_POSTGRESQL_HOST environment variable.
-    * The port for PostgreSQL is `3306` but can be changed through the CYGNUS_POSTGRESQL_PORT environment variable.
+    * The port for PostgreSQL is `5432` but can be changed through the CYGNUS_POSTGRESQL_PORT environment variable.
     * The user for PostgreSQL is `postgresql` but can be changed through the CYGNUS_POSTGRESQL_USER environment variable.
     * The pass for PostgreSQL is `postgresql` but can be changed through the CYGNUS_POSTGRESQL_PASS environment variable.
+    * The database for PostgreSQL is `postgres` but can be changed through the CYGNUS_POSTGRESQL_DATABASE environment variable.
```

Comment:
added a sentence to describe `CYGNUS_POSTGRESQL_DATABASE`.

Reply:
Ok. Thanks! NTC